stdlib/public/SDK/Foundation/NSCoder.swift

```diff
@@ -98,9 +72,9 @@ extension NSCoder {
 
   @nonobjc
   @available(OSX 10.11, iOS 9.0, *)
-  public func decodeTopLevelObject(forKey key: String) throws -> AnyObject? {
+  public func decodeTopLevelObject(forKey key: String) throws -> Any? {
```

Comment:
again another incorrect API change that should probably be restored and corrected for swift4